packages/dart/npt_flutter/lib/util/form_validator.dart

```diff
@@ -76,7 +89,7 @@ class FormValidator {
   static String? validateRemoteHostField(String? value) {
     final strings = AppLocalizations.of(App.navState.currentContext!)!;
     String valid =
-        r'^(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\.)+[a-zA-Z]{2,}|localhost|(?:\d{1,3}\.){3}\d{1,3}|(?:[a-fA-F0-9]{1,4}:){7}[a-fA-F0-9]{1,4})$';
+        r'^(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\.)*[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)$';
```

Comment:
This regex is unnecessarily nasty, I've done some verification of it, but there should be a comment:
`// matches an ip address, PQDN or FQDN`